### DISCUSSION TITLE
Relay-Code not working (anymore)?

### DISCUSSION THREAD

**Discussion by Athesis** [immense/Remotely#334] [Q&A]
Hello,

For some reason the Relay-Code is not working anymore. Is there any possibillity to check / renew this code? 

Thanks!

**Comment by bitbound** [comment]
I found the issue for this, and it should be fixed now on master.

It was only affecting the Windows installer, though.  So if you were seeing it elsewhere, let me know. 

Also, for anyone who may not know, the relay code only works if the clients are built from source and have the server URL embedded in them.  This can be done automatically using the `Remotely_Server_Installer` tool and selecting `no` for using pre-built packages.

**Reply by Athesis** [reply]
So, using the update-scripts won't update by itself?
